ConsentViewController/Classes/ConsentViewController.swift

```diff
@@ -274,12 +275,20 @@ import Reachability
                 return
             }
             webView.load(URLRequest(url: messageUrl))
+            messageStatus = .loaded
         } catch let error as ConsentViewControllerError {
+            messageStatus = .notStarted
```

Comment:
Do we need a similar step to this one, where you reset the `messageStatus` in the `Reachability` guard above? Otherwise if it errors due to network connection, we won't be able to retry!

Reply:
Good point. My idea, at first, wasn't to make it retriable, but simply to inform about the status of the message. But, in any case, we should reset `messageStatus` to `.notStarted` in the guard clause. 
Regarding  when `setSubjectToGDPR` throw an error, this will be captured by the `catch` and there we already reset the message status.